scripts/common/utilbinaries.sh

```diff
@@ -95,6 +97,11 @@ function get_patch_yaml() {
                 ;;
             kubernetes-version)
                 ;;
+            kurl-install-directory)
```

Comment:
if this is set should i print the prompt with the same in the join script? the load images task?

Reply:
Yes, that makes sense